### PR TITLE
fix: favors normalized when resizing images

### PR DESCRIPTION
**Pull request by dzucconi**
Closes: [GRO-1350](https://artsyproduct.atlassian.net/browse/GRO-1350)

Let's merge this on Monday to give the cache key changes to warm over the weekend, since this will blow away the cache again.

The major target here was the artwork grid item, portrait oriented works were particularly affected. But I tracked down a few more instances.

### CODE REVIEW

File: src/Apps/Collect/Routes/Collection/Components/Header/DefaultHeaderArtwork.tsx [hunk 1]
```diff
@@ -62,7 +62,11 @@ export const DefaultHeaderArtworkFragmentContainer = createFragmentContainer(
         href
         slug
         image {
-          resized(width: 300, height: 450) {
+          resized(
+            width: 300
+            height: 450
+            version: ["normalized", "larger", "large"]
```
Comment (jonallured) [inline src/Apps/Collect/Routes/Collection/Components/Header/DefaultHeaderArtwork.tsx:68]: TIL this can be an array??

Comment (dzucconi) [reply src/Apps/Collect/Routes/Collection/Components/Header/DefaultHeaderArtwork.tsx:68]: [Unfortunately.](https://github.com/artsy/metaphysics/blob/5f55ee918b29767d16fc915582195c5627455d6c/src/schema/v2/image/normalize.ts#L19-L35) — the image versions are relatively unreliable — the response might be missing a few.

**Comment by mzikherman** [comment]
conflix but lessgo

**Comment by dzucconi** [comment]
Closes in favor of https://github.com/artsy/force/pull/11333

**Pull request closed without merging**